identity/conf/routes

```diff
@@ -62,6 +62,7 @@ POST        /privacy/edit-ajax                      controllers.EditProfileContr
 GET         /email-prefs                            controllers.EditProfileController.displayEmailPrefsForm(consentsUpdated: Boolean ?= false, consentHint: Option[String])
 POST        /email-prefs                            controllers.EditProfileController.saveEmailPreferencesAjax
 
-GET         /consent                                controllers.EditProfileController.displayConsentJourneyForm(journey: Option[String], consentHint: Option[String], newsletterHint: Option[String])
-POST        /complete-consent                         controllers.EditProfileController.submitRepermissionedFlag
+GET         /consents                               controllers.EditProfileController.displayConsentJourneyForm(journey = "consent", consentHint: Option[String], newsletterHint: Option[String])
```

Comment:
it feels like these 2 routes are the same one (with optional journey and trailing slash) but not a big deal if fiddly to get working

Reply:
Yep it's true it seems like duplication and doesn't look very neat, but from what I can tell it's the only way to have the routes like this, as you can pass query params as optional but not actual /routes/.

Reply:
You could do it with regex, but not important